### PR TITLE
[New] Add instance id to response headers

### PR DESCRIPTION
**Pull request by geekgonecrazy**
@RocketChat/core 

This will help a lot with debugging multi-instance issues.

Debating putting behind a setting.  I don't think we use instance id for anything special, so I don't know that exposing it would be harmful in any way.

### CODE REVIEW

File: packages/rocketchat-lib/package.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ Npm.depends({
 
 Package.onUse(function(api) {
 	api.use('rate-limit');
+	api.use('webapp')
```
Comment (geekgonecrazy) [inline packages/rocketchat-lib/package.js]: fixed.

File: packages/rocketchat-lib/server/lib/debug.js [hunk 2]
```diff
@@ -36,3 +36,9 @@ Meteor.publish = function(name, func) {
 		return func.apply(this, arguments);
 	});
 };
+
+WebApp.rawConnectHandlers.use(function(req, res, next) {
+	const setHeader = res.setHeader;
```
Comment (geekgonecrazy) [inline packages/rocketchat-lib/server/lib/debug.js]: irrelevant on old commit.

File: packages/rocketchat-lib/server/lib/debug.js [hunk 3]
```diff
@@ -36,3 +36,9 @@ Meteor.publish = function(name, func) {
...
+	res.setHeader('X-Instance-ID', InstanceStatus.id());
```
Comment (geekgonecrazy) [inline packages/rocketchat-lib/server/lib/debug.js]: fixed :+1:

**Comment by sampaiodiego** [comment]
should we add a `[NEW]` badge?

**Comment by geekgonecrazy** [comment]
@sampaiodiego added!

**Pull request merged**